src/Command/ModuleRefresh.php

```diff
@@ -51,18 +44,19 @@ protected function configure()
     }
 
     /**
-     * {@inheritdoc}
+     * {@inheritDoc}
      *
-     * @throws ProcessException
+     * @throws ShellException
+     * @throws FileSystemException
      */
     public function execute(InputInterface $input, OutputInterface $output)
     {
-        try {
-            $this->process->execute();
-        } catch (ProcessException $exception) {
-            $this->logger->critical($exception->getMessage());
+        $enabledModules = $this->module->refresh();
```

Comment:
Was decided not to log command execution to cloud.log. if so, add this to note release too.

Reply:
It does not log to file. It just shows in the console.